Formula/coursier.rb

```diff
@@ -21,6 +21,9 @@ def install
     (libexec/"bin").install "coursier.jar"
     chmod 0755, libexec/"bin/coursier.jar"
     (bin/"coursier").write_env_script libexec/"bin/coursier.jar", Language::Java.overridable_java_home_env
+
+    chmod 0555, bin/"coursier"
```

Comment:
Why this? That should be managed by `write_env_script`

Reply:
Without this, executing next line fails.

Reply:
Why does it fail?

Reply:
permission denied:

`bin/"coursier"` seems not to have enough permission to be executed.

```
cazador@crystal coursier-20220922-16369-12h09lh % ls -l /usr/local/Cellar/coursier/2.1.0-M6_1/bin/coursier
-rw-r--r--  1 cazador  admin  171 Sep 22 11:42 /usr/local/Cellar/coursier/2.1.0-M6_1/bin/coursier
cazador@crystal coursier-20220922-16369-12h09lh % /usr/local/Cellar/coursier/2.1.0-M6_1/bin/coursier completions bash
zsh: permission denied: /usr/local/Cellar/coursier/2.1.0-M6_1/bin/coursier
```

Reply:
I've seen this too where the `write_env_script` lacks execute permissions during `install` but has them afterward (e.g. `post_install` or `test`). May be something worth looking into in brew as we probably don't want to manually change permissions on file that brew should be handling.

Reply:
Also, rather than changing permissions, may just want to run it using `bash` rather than relying on shebang.